src/style-spec/reference/v8.json

```diff
@@ -2524,6 +2500,72 @@
     "value": "*",
     "doc": "A filter selects specific features from a layer."
   },
+  "filter_symbol": {
+    "type": "boolean",
+    "doc": "Expression used to determine whether a symbol is being displayed or not. Symbols support `dynamic-filtering`, so this expression can account for the current state of the view by incorporating `[\"pitch\"]` and `[\"distance-from-center\"]` as part of the expression.",
+    "default": false,
+    "transition": false,
+    "property-type": "data-driven",
+    "expression": {
+      "interpolated": false,
+      "parameters": ["zoom", "feature", "pitch", "distance-from-center"]
+    }
+  },
+  "filter_fill": {
+    "type": "boolean",
+    "doc": "Expression used to determine whether a polygon is being displayed or not. Fill layer does NOT support `dynamic-filtering`, so this expression can NOT account for the current state of the view by incorporating `[\"pitch\"]` and `[\"distance-from-center\"]` as part of the expression.",
+    "default": false,
+    "transition": false,
+    "property-type": "data-driven",
+    "expression": {
+      "interpolated": false,
+      "parameters": ["zoom", "feature"]
+    }
+  },
+  "filter_line": {
+    "type": "boolean",
+    "doc": "Expression used to determine whether a LineString or a Polygon is being displayed or not. Line layer does NOT support `dynamic-filtering`, so this expression can NOT account for the current state of the view by incorporating `[\"pitch\"]` and `[\"distance-from-center\"]` as part of the expression.",
```

Comment:
Ditto, but I'll return to the rewording once we've nailed down the analogous wording for at least one property.